tests/test_simulation.py

```diff
@@ -611,7 +632,67 @@ def test_dephasing():
         )
 
 
-def test_add_config():
+def test_depolarizing():
+    np.random.seed(123)
+    reg = Register.from_coordinates([(0, 0)], prefix="q")
+    seq = Sequence(reg, Chadoq2)
+    seq.declare_channel("ch0", "rydberg_global")
+    duration = 2500
+    pulse = Pulse.ConstantPulse(duration, np.pi, 0, 0)
+    seq.add(pulse, "ch0")
+    sim = Simulation(
+        seq, sampling_rate=0.01, config=SimConfig(noise="depolarizing")
+    )
+    assert sim.run().sample_final_state() == Counter({"0": 587, "1": 413})
```

Comment:
Check also that the state is not pure anymore, otherwise you can't distinguish between a mixed state and an equal superpostion

Reply:
I took the trace of the square of the density matrix at the end of the evolution and checked if it's lower than one without being close to one.

Reply:
Correct me if I'm wrong, but it should actually be close to `1-3p/4`, no? Perhaps you can check for that

Reply:
Is it enough? Because I made the same test for the dephasing channel and the test below worked. Does it mean the state is a mixed state?

Reply:
The absolute difference is 0.03, is it close enough?

Reply:
Are you struggling with this?

Reply:
Yes. I don't understand why `1- 3*P`. With `rho = p/2 I + (1-p) 1` I found `tr(rho**2) = 2 (1-3/4p) ** 2 + 3/8 p**2`

Reply:
By expanding `rho**2` and using `tr(sigma_i) = 0` and `tr(sigma_i * sigma_j) = 0` for `i!=j`